tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -232,3 +235,41 @@ def test_nodereplacement_proactive(self):
         assert ceph_cluster_obj.wait_for_rebalance(
             timeout=1800
         ), "Data re-balance failed to complete"
+
+
```

Comment:
Please also create a test case and add the polarion ID (@pytest.mark.polarion_id(<Polarion_ID>)

Reply:
Done